src/main/java/com/ra4king/circuitsim/gui/CircuitManager.java

```diff
@@ -361,16 +364,23 @@ public void paint() {
 			graphics.setFont(GuiUtils.getFont(13));
 			graphics.setFontSmoothingType(FontSmoothingType.LCD);
 			
-			graphics.setFill(Color.LIGHTGRAY);
+			boolean drawGrid = showGrid.getValue();
+			if (drawGrid) {
```

Comment:
Prefer a ternary within the setFill:

`graphics.drawFill(drawGrid ? Color.LIGHTGRAY : Color.WHITE)